syscalls.c

```diff
@@ -1129,12 +1312,21 @@ int syscalls_sys_accept4(char *ustack)
 	GETFROMSTACK(ustack, socklen_t *, address_len, 2);
 	GETFROMSTACK(ustack, int, flags, 3);
 
+	if (vm_mapBelongs(proc, address, sizeof(*address)) < 0) {
```

Comment:
`address` is optional. If not NULL it's size should be passed in `*address_len`.